FantasmoSDK/Classes/FMLocationManager.swift

```diff
@@ -72,9 +72,12 @@ open class FMLocationManager: NSObject, FMApiDelegate {
     private var lastFrame: ARFrame?
     private var lastLocation: CLLocation?
     
-    private weak var delegate: FMLocationDelegate?
+    private var delegate: FMLocationDelegate?
+    private var isConnectedToServer: Bool = false
```

Comment:
What benefit does this provide? Sanity check for implementers? The name could be better, as we are not connected to a server.

Reply:
Yes, I agree. I had this thought concerning "connectedness"  when adding this property.

Effectively what we do is initialization of SDK rather than make connection to it. "Connection" word is intended for something remote. I suggest that we rename "connect" -> "initialize" and rename "isConnectedToServer" -> "isSdkInitialized".

Do you agree?